model/masl/models/masl/maslpopulation/file/file.xtuml

```diff
@@ -81,77 +76,57 @@ elif ( "domain" == element )
       activities = domain_routines;
       for each activity in activities
         select one file related by activity->markable[R3783]->element[R3786]->file[R3795];
-        select one code_block related by activity->code_block[R3794];
         if ( not_empty file )
           destination = file.getname();
           // Domain files go into a directory named after the domain.
           file.directory( name:domain.name );
-          if ( empty code_block )	// generate a file stub
-            select one class_op related by activity->operation[R3704];
-            select one deferral related by class_op->subsuper[R3793];
-            if ( empty class_op or ( not_empty class_op and empty deferral ) )
-              file.gen( destination:destination );
-            end if;
-          else						// activity exists, copy it
-            file.copy( source:code_block.sourcefile, destination:destination );
-          end if;
+		  select one class_op related by activity->operation[R3704];
```

Comment:
Ack, barf... tabs?!?  You don't have to change it in this file, but please change it in your editor for future files.